configure.ac

```diff
@@ -219,7 +219,7 @@ else
 	AC_MSG_RESULT(no)
 fi
 
-AC_CHECK_HEADERS([atomic.h siginfo.h sys/int_limits.h sys/lwp.h signal.h sys/byteorder.h poll.h sys/poll.h sys/varargs.h stdint.h termio.h stropts.h sys/ttycom.h wait.h alloca.h sys/sendfile.h sys/types.h linux/unistd.h sys/ioctl.h sys/uio.h])
+AC_CHECK_HEADERS([stdatomic.h siginfo.h sys/int_limits.h sys/lwp.h signal.h sys/byteorder.h poll.h sys/poll.h sys/varargs.h stdint.h termio.h stropts.h sys/ttycom.h wait.h alloca.h sys/sendfile.h sys/types.h linux/unistd.h sys/ioctl.h sys/uio.h])
```

Comment:
Instead of removing `atomic.h`, why not add `stdatomic.h`?  Then, based on [1], you'd have either `HAVE_STDATOMIC_H` or `HAVE_ATOMIC_H` macros defined.

[1] https://www.gnu.org/software/autoconf/manual/autoconf-2.67/html_node/Generic-Headers.html